toolkit/docs/formats/imageconfig.md

```diff
@@ -212,6 +212,25 @@ A sample PreInstallScripts entry pointing to two install scripts where one has i
 ],
 ```
 
+### Networks
+
+Kickstart style unattended installations tends to perform network configurations during the installation process to set IP address, configure the hostname, DNS etc. Thus, the `Networks` entry is added to enable the users to specify the network config parameters. Currently, the Mariner tooling only supports a subset of the kickstart network command options: `bootproto`, `gateway`, `ip`, `net mask`, `DNS` and `device`. Hostname can be configured using the `Hostname` entry of the image config. 
+
+A sample Networks entry pointing to one network configuration:
+``` json
```

Comment:
I think you've not accidentally removed the closing three back quotes.

Reply:
Fixed